extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -682,13 +682,43 @@ export class ConsentUI {
         (this.restrictFullscreenOn_ &&
           this.document_.activeElement !== this.ui_)
       ) {
-        // TODO (@torch2424) Send response back if enter fullscreen was succesful
+        const error = 'Could not enter fullscreen';
```

Comment:
nit: Let's call it errorMsg, so we know that the type here is a string, not an Error